ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__nightly-4.11-upgrade-from-stable-4.10-from-stable-4.9.yaml

```diff
@@ -62,6 +62,31 @@ tests:
           cpu: 100m
           memory: 200Mi
     - chain: cucushift-upgrade
+- as: vsphere-ipi-disconnected
+  cron: 0 22 */3 * *
```

Comment:
Perhaps we can test this kind of upgrade less often. E.g, once every week

Reply:
sure. let me update. Thanks for the suggestion!